props/saber_BC_buttons.h

```diff
@@ -1968,6 +1968,24 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
 
 #endif  // BC_DUAL_BLADES
 
+  bool Parse(const char *cmd, const char* arg) override {
```

Comment:
Don't think this belongs in this PR.

Reply:
I commented about this I swear. Don't know where it went.
THis was in a mode initially, then moved to the body of the BCSaberProp class.

I figured it applied here as it's "adding serial commands"

If you want I can make another standalone PR for it.... ?

Reply:
Please make separate PR